modules/platforms/cpp/ignite/client/transaction/transaction_options.h

```diff
@@ -0,0 +1,48 @@
+/*
+* Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#pragma once
+
+/**
+ * Transaction options.
+ */
+class transaction_options {
+public:
+    transaction_options() = default;
+    ~transaction_options() = default;
+    transaction_options(const transaction_options&) = default;
+    transaction_options(transaction_options &&) = default;
+    transaction_options &operator=(const transaction_options &other) = default;
+    transaction_options &operator=(transaction_options &&other) = default;
```

Comment:
```suggestion
    transaction_options() = default;
```
It's a good practice to not define anything you don't really have to. You only have to define copy and move operations if you define a destructor, but there is no need for you to explicitly declare a destructor if it's default anyway.